app/src/main/java/io/pslab/fragment/AboutUsFragment.java

```diff
@@ -66,7 +65,7 @@ public boolean onOptionsItemSelected(MenuItem item) {
         return super.onOptionsItemSelected(item);
     }
 
-    private void simulateDayNight(int currentSetting) {
+/*    private void simulateDayNight(int currentSetting) {
```

Comment:
Rather than commenting out, let's remove this section. If we want to refer back, we can always use git history

Reply:
Done @CloudyPadmal